src/scss/_objects.resources-tile.scss

```diff
@@ -0,0 +1,12 @@
+.resources-tile {
+  background-color: #fff;
+
+  &__thumbnail {
+    width: 100px;
+    height: 100px;
+  }
+
+  .heading--uppercase {
+    letter-spacing: normal;
```

Comment:
I've noticed that you're overriding the letter-spacing for the uppercase heading in several places.  Is this something that has been discussed with the designers?  In the long run, I think it would be preferable to either get the designers to agree to the existing letter-spacing as it currently is on Grommet, or talk to them about getting the existing letter-spacing style removed from Grommet, so we're not ending up with inconsistencies between the toolkit and any other Grommet elements that could hypothetically be used on the same page.